interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+    deny /sys/devices/virt[^u]*{,/**} wklx,
+    deny /sys/devices/virtu[^a]*{,/**} wklx,
+    deny /sys/devices/virtua[^l]*{,/**} wklx,
+    deny /sys/devices/virtual/[^n]*{,/**} wklx,
+    deny /sys/devices/virtual/n[^e]*{,/**} wklx,
+    deny /sys/devices/virtual/ne[^t]*{,/**} wklx,
+    deny /sys/devices/virtual/net?*{,/**} wklx,
+    deny /sys/devices/virtual?*{,/**} wklx,
+    deny /sys/devices?*{,/**} wklx,
+    deny /sys/f[^s]*{,/**} wklx,
+    deny /sys/fs/[^c]*{,/**} wklx,
+    deny /sys/fs/c[^g]*{,/**} wklx,
+    deny /sys/fs/cg[^r]*{,/**} wklx,
+    deny /sys/fs/cgr[^o]*{,/**} wklx,
+    deny /sys/fs/cgro[^u]*{,/**} wklx,
+    deny /sys/fs/cgrou[^p]*{,/**} wklx,
+    deny /sys/fs/cgroup?*{,/**} wklx,
+    deny /sys/fs?*{,/**} wklx,
+
+    ### Feature: unix
+    # Allow receive via unix sockets from anywhere
+    unix (receive),
+
+    # Allow all unix in the container
+    unix peer=(label=@{profile_name}),
+
+    ### Feature: cgroup namespace
+    mount fstype=cgroup -> /sys/fs/cgroup/**,
+
+    ### Configuration: nesting
+    pivot_root,
+    ptrace,
+    signal,
+
+    deny /dev/.lxd/proc/** rw,
+    deny /dev/.lxd/sys/** rw,
+
+    mount /var/lib/lxd/shmounts/ -> /var/lib/lxd/shmounts/,
+    mount none -> /var/lib/lxd/shmounts/,
+    mount fstype=proc -> /usr/lib/*/lxc/**,
+    mount fstype=sysfs -> /usr/lib/*/lxc/**,
+    mount options=(rw,bind),
+    mount options=(rw,rbind),
+    mount options=(rw,make-rshared),
+
+    # there doesn't seem to be a way to ask for:
+    # mount options=(ro,nosuid,nodev,noexec,remount,bind),
+    # as we always get mount to $cdir/proc/sys with those flags denied
+    # So allow all mounts until that is straightened out:
+    mount,
+    mount options=bind /var/lib/lxd/shmounts/** -> /var/lib/lxd/**,
+
+    mount options=(rw,make-slave) -> **,
+    mount options=(rw,make-rslave) -> **,
+    mount options=(rw,make-shared) -> **,
+    mount options=(rw,make-rshared) -> **,
+    mount options=(rw,make-private) -> **,
+    mount options=(rw,make-rprivate) -> **,
+    mount options=(rw,make-unbindable) -> **,
+    mount options=(rw,make-runbindable) -> **,
+
+    mount options=(rw,bind),
+    mount options=(rw,rbind),
+}
+`
+
+const anboxSupportConnectedPlugSecComp = `
+# On first startup the container manager needs to ensure that certain directory are
+# own by the UID range we assign to the Android container.
+chown
+
+# Various things get bind mount into Android container
+mount
+umount2
+
+# liblxc creates various device nodes in the /dev it sets up for the Android container
+mknod - |S_IFCHR -
+mknodat - - |S_IFCHR -
+setgroups
+sethostname
+pivot_root
+
+# needed?
+setns
+
+# only saw these once with aa-exec -p unconfined -- ...
+fchown
+fchown32
+fchownat
+setpriority
+
+# Needed by liblxc
+prctl PR_CAP_AMBIENT PR_CAP_AMBIENT_RAISE - - -
+`
+
+var anboxSupportConnectedPlugUDev = []string{
+	`KERNEL=="loop-control"`,
+	`KERNEL=="loop*", SUBSYSTEM=="block"`,
```

Comment:
Interesting that this ends up tagging all the snaps' squashfs files. I think that warrants a comment up above:
```
# This is an information leak as adds all the snaps squashfs images to the
# device cgroup.